homeassistant/components/synology/camera.py

```diff
@@ -108,6 +131,8 @@ def update(self):
         self._motion_setting = self._surveillance.get_motion_setting(
             self._camera.camera_id)
         self.is_streaming = self._camera.is_enabled
+        self.hass.states.set('camera.synology_home_mode',
```

Comment:
No no no. 

Make this a binary sensor platform with a binary sensor entity. This is not allowed.